neurovault/apps/statmaps/views.py

```diff
@@ -165,6 +167,11 @@ def view_image(request, pk, collection_cid=None):
     user_owns_image = True if owner_or_contrib(request,image.collection) else False
     api_cid = pk
 
+    # Comparison is possible if pk is in matrix columns
+    corr_df = pandas.read_pickle(os.path.abspath(os.path.join(os.path.dirname( neurovault.settings.BASE_DIR ), '../..', 'image_data/matrices/pearson_corr.pkl')))
```

Comment:
this will still fail if the private media path in settings changes (for example in a different installation) - better use the variable from settings